src/Utilities/Callbacks/Callbacks.jl

```diff
@@ -63,7 +61,7 @@ Update
                 )
                 isnan(energy) && error("norm(Q) is NaN")
             end
-            user_info_callback(init)
+            user_info_callback()
```

Comment:
Requires [this](https://github.com/CliMA/ClimateMachine.jl/blob/master/test/Driver/gcm_driver_test.jl#L110) and [this](https://github.com/CliMA/ClimateMachine.jl/blob/master/test/Driver/driver_test.jl#L139) to be fixed.